interfaces/builtin/optical_drive.go

```diff
@@ -28,27 +37,73 @@ const opticalDriveBaseDeclarationSlots = `
         - core
```

Comment:
To flag for manual connection when write is true, the base declaration needs to be updated to be:
```
const opticalDriveBaseDeclarationSlots = `
  optical-drive:
    allow-installation:
      slot-snap-type:
        - core
    deny-connection:
      plug-attributes:
        write: true
    deny-auto-connection:
       plug-attributes:
         write: true
`
```

Reply:
@jdstrand The default of auto-connection is already false, right? This snippet makes it look like it won't auto-connect in either case. I'm probably missing something again about the details here.

Reply:
@niemeyer - the base declaration prior to this PR allows auto-connection because there are no constraints (originally discussed here: https://github.com/snapcore/snapd/pull/1368#discussion_r67888452). The base declaration change I suggested continues to allow auto-connection *except* when specifying 'write: true'.